.github/workflows/config/mlc_config.json

```diff
@@ -22,6 +22,12 @@
             "pattern": "https://cydia.akemi.ai/"
         },{
             "pattern": "https://cydia.angelxwind.net"
+        },
+        {
+            "pattern": "^https://developers.android.com"
```

Comment:
why using `^` on these ones?

Reply:
When it finds or looks for links, I want it to find a link that start with `https://......`. An opposite example would be `https://google.com?<somevariable>=https://developers.android.com` and I don't want that to happen.

Reply:
Removed the `^` to see if the markdown link checker would take it. For some reason it's not working with the updated config.